### PR TITLE
feat/auth

### PR DESCRIPTION
**Pull request by wijdanepitech**

### CODE REVIEW

File: src/pages/Register.tsx [hunk 1]
```diff
@@ -1,5 +1,91 @@
+import React from 'react';
```
Comment (Fautour) [inline src/pages/Register.tsx:1]: You need to remove this line and after that we can merge

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -1,5 +1,84 @@
+import React from 'react';
```
Comment (Fautour) [inline src/pages/Login.tsx:1]: You need to remove this line and after that we can merge

File: src/pages/Register.tsx (hunk 1 above)
Comment (Fautour) [inline src/pages/Register.tsx:1]:
```suggestion
```

File: src/pages/Login.tsx (hunk 2 above)
Comment (Fautour) [inline src/pages/Login.tsx:1]:
```suggestion
```

**Review by Fautour — CHANGES_REQUESTED** [review]

**Comment by Lxdovic** [comment]
branch moves to feat/auth see #56

**Pull request closed without merging**